src/Illuminate/Console/Scheduling/ScheduleFinishCommand.php

```diff
@@ -31,12 +33,17 @@ class ScheduleFinishCommand extends Command
      * Execute the console command.
      *
      * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
+     * @param  \Illuminate\Contracts\Events\Dispatcher  $dispatcher
      * @return void
      */
-    public function handle(Schedule $schedule)
+    public function handle(Schedule $schedule, Dispatcher $dispatcher)
```

Comment:
This is a breaking change unfortunately.

Reply:
Oh oops :-( 
Thanks for that. I made the change in this way as it follows the pattern in the original PR I referenced https://github.com/laravel/framework/pull/29888 which was accepted into the currently active version. 

Happy to make the Dispatcher::class instance using the app/resolve helper inside the handle function instead to avoid breaking the handle method's API?

Reply:
Think it's best that this targets master instead

Reply:
Fair call. Thanks, will raise again targetting master tomorrow.